### ISSUE TITLE
Xcode 14.3 build issue

### ISSUE THREAD

**Issue by pyby** [urbanairship/ios-library#350]
# Preliminary Info

### What Airship dependencies are you using?
iOS library 16.10.7. 
With SPM dependency tool.

### What are the versions of any relevant development tools you are using?
Xcode 14.3 beta 1, running on Mac OS Ventura 13.2.1.
https://developer.apple.com/documentation/xcode-release-notes/xcode-14_3-release-notes

# Report

### What unexpected behavior are you seeing?
The project does not build anymore, with a swift error on `Shapes` swift structure.

### What is the expected behavior?
Build passed with the new Xcode 14.3 version.

### What are the steps to reproduce the unexpected behavior?
Open the sample project (`Sample/Sample.xcodeproj`) with Xcode 14.3 beta 1.
Try to build it.

### Do you have logging for the issue?

In `Shapes.swift` file:
```
Line 30:
if let strokeColor = strokeColor, strokeWidth > 0 {

Error: Initializer for conditional binding must have Optional type, not 'Color'
```

Xcode 14.3 checks now the optionality of each condition in SwiftUI views.

**Comment by oristanovic** [comment]
Thanks for the report, we're taking a look

**Comment by MacMark** [comment]
I have this issue too. It prevents running the Airship lib with the latest iOS beta because this need Xcode 14.3.

**Comment by rlepinski** [comment]
We have it fixed internally and working on getting a release out.

**Comment by rlepinski** [comment]
Fixed in 16.11.0
